doc/source/generation.rst

```diff
@@ -69,7 +83,9 @@ Lattice
 
 creates a 3x3 grid in two dimensions (9 vertices total). `circular` is used to connect each edge of the lattice back onto the other side, a process also known as "periodic boundary condition" that is sometimes helpful to smoothen out edge effects.
 
-.. note:: The line graph is a one dimensional, non-circular lattice.
+The one dimensional, circular case (a ring) is important enough to deserve its own constructor :meth:`Graph.Ring`.
+
+.. note:: The line graph is a one dimensional, non-circular lattice, and can be also constructed as a non-circular ring.
```

Comment:
Maybe it's worth removing this paragraph completely; it will cause confusion as `Graph.line_graph()` is a separate method that constructs the [line graph](https://www.wikiwand.com/en/Line_graph) of another graph, and here "line graph" is used as "a graph that looks like a line"

Reply:
A "one-dimensional lattice" is called a "path graph", and commonly denoted by $P_n$ by graph theorists, $n$ being the number of vertices.

Reply:
Yep, I wasn't sure so it's good you caught it.